frontend/src/lib/nock-util.ts

```diff
@@ -46,9 +48,8 @@ export function nockGet<Resource extends IResource>(
     statusCode = 200,
     polling = true
 ) {
-    const nockScope = nocked(process.env.JEST_DEFAULT_HOST as string, { encodedQueryParams: true }).get(
-        getResourceNameApiPath(resource)
-    )
+    const resourcePath = getResourceNameApiPathTestHelper(resource)
+    const nockScope = nock(process.env.JEST_DEFAULT_HOST as string, { encodedQueryParams: true }).get(resourcePath)
```

Comment:
Did you mean to change `nocked` to `nock` or is this a bad merge? Note - there are multiple instances of this.